hazelcast/src/main/java/com/hazelcast/nio/serialization/Data.java

```diff
@@ -113,4 +113,10 @@
      */
     boolean isPortable();
 
+    /**
+     * Converts Data to HeapData. Useful for offheap conversion.
+     *
+     * @return the onheap representation of data. If data is null, null is returned.
```

Comment:
Data can't be null :) Otherwise you would get a NPE.

Reply:
remove from the doc.